web-components/src/components/tabs/Tab.ts

```diff
@@ -240,6 +240,14 @@ export namespace Tab {
           <slot class="tab-slot"></slot>
           ${this.isCrossVisible && this.closable ? this.renderCrossButton() : ""}
         </div>
+        <!-- Invisible button for legacy third party test compatibility -->
+        <button
+          type="button"
```

Comment:
Third party test is checking the tabs shadow dom for a button. add a hidden button to keep tests passing until fixed